agent/agent.go

```diff
@@ -149,6 +167,13 @@ func (a *orbAgent) Stop() {
 		}
 	}
 	a.client.Disconnect(250)
+	defer close(a.hbDone)
```

Comment:
does the defer have an effect here since the function is ending?

Reply:
Yeah, I went by the book on closing channels with any concurrent issue, this actually ensure that the channel doesn't stay opened.